docs/installation/sources/sources_linux.rst

```diff
@@ -175,6 +175,13 @@ This section explains how to use it to compile *eProsima Fast DDS* and its depen
 
        pip3 install -U colcon-common-extensions vcstool
 
+#. After this step colcon and vcstool will be in $HOME/.local/bin and have to be added in the PATH.
+   For example, on Ubuntu use:
+
+   .. code-block:: bash
+
+       echo "export PATH="$HOME/.local/bin:$PATH"" >> ~/.bashrc
```

Comment:
On Ubuntu 20.04, `$HOME/.local/bin` is already in the `PATH`

Reply:
I managed to get the following warning on a Focal docker:

`WARNING: The script colcon is installed in '/home/test2/.local/bin' which is not on PATH.
  Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.`
  
Maybe it's worth it to reword this step and ask the user to check his PATH for this folder presence and update it if it is not there.